src/requirements/data/templates/ece.ts

```diff
@@ -28,16 +28,28 @@ const eceTemplate: Template = [
   },
   {
     reqGroup: 'EN',
+    name: 'Intro to Engineering',
+    placeholderSemesters: [1],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Computing from engineering',
```

Comment:
Sorry, should have been more specific - this requirement should just be called "Computing" !